style.css

```diff
@@ -414,3 +414,4 @@ body {
     width: 50%;
   }
 }
+
```

Comment:
Takové rozdílné jednotky se nepoužívají... zkusil bych u obou 12% jestli to není OK :+1: 

```
padding-right: 13%;
padding-left: 11%;
```

Ř. 78:

Taktéž mi to přijde jako blbost, použil bych rem místo px.

```
padding-left: 250px;
padding-right: 250px;
```

Alespoň pro barvičky bych používal proměnné v CSS, viz. zde:
https://www.w3schools.com/css/css3_variables.asp
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

Toto např. jde zjednodušit (ř. 286):

```
.content {
    padding-left: 100px;
    padding-right: 100px;
  }
  .topnav {
    padding-left: 100px;
    padding-right: 100px;
  }
```
na
```
.content, .topnav {
    padding: 0 100px;
  }
``` 

Selecktory pod řádkem 295 jdou také sloučit do jednoho..
343 ty fontsize také :+1: 

Styly vypadají dost dobře jinak :100: :1st_place_medal:

Reply:
Teď mě tak ještě nepadlo, že ten stylesheet už je dost dlouhý.. taky by už dávalo smysl jej rozdělit do css souborů podle toho, k čemu se dané selektory používají :wink: